third_party/libxml/BUILD.gn

```diff
@@ -6,9 +6,10 @@
 # headers.  These were generated by running the configure script offline.
 if (is_starboard) {
```

Comment:
We've update `is_starboard` 3p conditionals to conditionals on `use_cobalt_customizations`—would you be able to update the conditionals in this file? Alternatively fine to do as a follow up change

Reply:
done